src/elements/kw-social-comment/kw-social-comment.html

```diff
@@ -0,0 +1,194 @@
+                font-family: "Bariol";
+                color: var(--color-grey-dark);
+                width: 100%;
+            }
+            
+            .content p b {
+                margin-right: 6px;
+            }
+            
+            .content p {
+                margin-top: 6px;
+                margin-bottom: 6px;
+            }
+            
+            .content date-util {
+                font-size: 14px;
+                font-family: "Bariol";
+                color: var(--color-grey-dark);
+            }
+            
+            .comments-box p {
+                font-family: "Bariol";
+                color: var(--color-grey-dark);
+                font-size: 16px;
+            }
+            
+            iron-scroll-threshold {
+                @apply --layout-flex;
+                @apply --layout-vertical;
+            }
+        </style>
+        <div class="comments-box">
+            <h3>Be the first to comment!</h3>
+            <div class="line"></div>
+            <template is="dom-repeat" items="{{comments}}">
+                <div class="comment">
+                    <div class="comment-avatar"><img src="/assets/icons/shares/commentAvatar.svg" /></div>
+                    <div class="content">
+                        <p><b>{{item.author.username}}</b>{{item.text}}</p>
+                        <date-util format="H:mm , dd Mon yyyy" date="[[_createDate(item.date_created)]]"></date-util>
+                    </div>
+                </div>
+            </template>
+            <div class="line"></div>
+            <div class="inputcomment">
+                <div class="comment-avatar"><img src="/assets/icons/shares/commentAvatar.svg" /></div>
+                <div class="commentbox">
+                    <input type="text" placeholder="Type your message ...">
+                    <button type="submit" class="orange-button comment-button" onclick="">comment <img src="/assets/icons/shares/comment.svg" /></button>
+                </div>
+            </div>
+            <p>Remember to be nice</p>
+        </div>
+        </kw-share-shell>
+    </template>
+    <script>
+        Polymer({
+            is: 'kw-social-comment',
+            behaviors: [Kano.APIClient],
+            properties: {
+                comments: {
+                    type: Array,
+                    value:[]
+                    
+                },
+                share: {
+                    type: Object
+                }
+            },
+            attached () {
+                this.page = 0;
+                this.loadData();
+            },
+            loadData(){
+                // When we hit the last page
+                if (!Boolean(this.share)) {
+                    return;
+                }
+                fetch(this._getUrl('comments', { type: this.share.type ,id: this.share.id}))
+                .then(r => r.json())
+                .then(r => {
+                    r.entries.forEach(comment => {
```

Comment:
not the right indenting, but also, I would pass the comments from the outside and keep this element as a display element. I don't think the API call should happen here and also the 'when we hit the last page' comment is a bit confusing. Maybe @pazdera can advise